README.md

```diff
@@ -41,6 +41,13 @@ FingerprintJS Pro Svelte SDK is an easy way to integrate [Fingerprint Pro](https
 
 This package works with Fingerprint Pro, it is not compatible with open-source FingerprintJS. See our documentation to learn more about the [difference between Fingerprint Pro and the open-source FingerprintJS](https://dev.fingerprint.com/docs/pro-vs-open-source).
```

Comment:
Have you considered moving this to the end the of "Requirements"? I am inclined to have it there, it does clarify a requirement of having a commercial Fingerprint account.

https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#40df07e276694e3bba6c0c532690cdc2